src/core/abstracts/Batch.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity >=0.8.22;
+
+import { Errors } from "../libraries/Errors.sol";
+
+/// @title Batch
+/// @notice This contract implements logic to batch call any function.
+/// @dev Forked from: https://github.com/boringcrypto/BoringSolidity/blob/master/contracts/BoringBatchable.sol
+abstract contract Batch {
+    /*//////////////////////////////////////////////////////////////////////////
+                         USER-FACING NON-CONSTANT FUNCTIONS
+    //////////////////////////////////////////////////////////////////////////*/
+
+    /// @notice Allows batched call to self, `this` contract.
+    /// @param calls An array of inputs for each call.
+    function batch(bytes[] calldata calls) external {
```

Comment:
This function should also be a part of `ISablierLockup` interface. We forgot to do the same with Flow.

Reply:
agree

Reply:
That works as well.